content/working-groups/devices.md

```diff
@@ -2,13 +2,21 @@
 title: WG-Devices
 type: docs
 weight: 20
+draft: true
 ---
 
```

Comment:
We are looking for leads for this working group. If you are interested, please [email the admin team](mailto:bbqs-admin@brain-bbqs.org).